edge/discussion_materials/onnx-edge-scope and profiling.md

```diff
@@ -0,0 +1,75 @@
+# ONNX Edge scope and profiling
+Author: Yedong Liu
+
+## Scope of Edge
+
+Edge computing is computing that’s done at or near the source of the data, instead of relying on the cloud at one of a dozen data centers to do all the work. Edge computation is largely or completely performed on distributed edge devices. The target of edge computing is any application or general functionality needing to be closer to the source of the action where system can process and interact with the real world instantly.
+
+
+## Edge scenario
+
+Computation offloading for real-time applications, such as facial recognition, smart city, intelligent traffic control, industrial IoT, video analysis and monitoring, border control, unattended shops, auto pilot and smart drones are all considered within edge computing scope. 
+
+Edge provides lower latency and reduces transmission costs, and edge computing still varies is highly scenario dependent. 
+
+In one of the scenarios, specifically for IoT devices, data comes in from the physical world via various sensors, and actions are taken to change physical state via various forms of output; by performing analytics and generating output directly at the site, communications bandwidth between edge and cloud is reduced. Edge computing takes advantage of proximity to the physical items of interest and also exploits the relationships those items may have to each other.
+
+In another scenario which requires low latency like auto pilot cars. A notable applications include online gaming. Game servers are running in the cloud and the rendered video is transferred to lightweight clients such as AR devices (mobile phone), VR glasses, etc. Conventional cloud games may suffer from high latency and insufficient bandwidth. As real-time games have strict constraints on latency, processing game simulation at the edge node is necessary for the immersive game plays.
+
+
+## Edge device
+
+An edge device is a device which provides an entry point into enterprise or service provider core networks. Examples include routers, routing switches, terminals and other devices. Edge devices also provide connections into carrier and internet service provider networks.
+
+In general, edge devices are normally routers and terminals that provide authenticated access to faster, more efficient backbone and core networks. Edge device is usually smart and has access to the core devices, often include Quality of Service (QoS) and multi-service functions to manage different types of traffic.
```

Comment:
"Edge device is usually smart and has access to the core devices, often include Quality of Service (QoS) and multi-service functions to manage different types of traffic."
I am not sure if this sentence is relevant/needed  here. What are core devices ?